File: server/bcephal-grid/src/main/java/com/moriset/bcephal/grid/domain/GrilleEditedElement.java
/**
 * 
 */
package com.moriset.bcephal.grid.domain;

import java.math.BigDecimal;
import java.util.Date;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.moriset.bcephal.utils.JsonDateDeserializer;
import com.moriset.bcephal.utils.JsonDateSerializer;

import lombok.Data;

/**
 * @author Moriset
 *
 */
@Data
public class GrilleEditedElement {

	private Long id;

	private GrilleColumn column;

	private String stringValue;

	private BigDecimal decimalValue;

	@JsonSerialize(using = JsonDateSerializer.class)
	@JsonDeserialize(using = JsonDateDeserializer.class)
	private Date dateValue;

	private Grille grid;
	
}

File: client/bcephal-client-model/Grids/GrilleEditedElement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Grids
{
    public class GrilleEditedElement
    {

        public long? Id { get; set; }

        public GrilleColumn Column { get; set; }

        public string StringValue { get; set; }

        public decimal? DecimalValue { get; set; }

        public DateTime? DateValue { get; set; }

        public Grille Grid { get; set; }

    }
}
